exchanges/exmo/exmo_wrapper.go

```diff
@@ -222,7 +217,7 @@ func (e *EXMO) UpdateTickers(ctx context.Context, a asset.Item) error {
 
 	for symbol, tick := range result {
 		var pair currency.Pair
-		pair, err = avail.DeriveFrom(strings.Replace(symbol, "_", "", 1))
+		pair, err = e.MatchSymbolWithAvailablePairs(symbol, a, true)
 		if err != nil {
 			return err
```

Comment:
Its hidden by `UpdateTradablePairs` in `TestMain`, but this also has the same potential issue:
    exmo_test.go:513: pair not found for exmusdt spot
    exmo_test.go:524: no tickers associated with base currency BCH etc

Any err returned from the new funcs in this ticker stuff needs to be looked at